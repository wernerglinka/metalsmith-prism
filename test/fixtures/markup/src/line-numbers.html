<pre>
  <code class="language-javascript">
    var name = "Rob";
  </code>
</pre>
