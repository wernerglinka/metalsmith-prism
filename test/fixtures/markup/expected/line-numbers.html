<html><head></head><body><pre class="language-javascript line-numbers"> <code class="language-javascript"> <span class="token keyword">var</span> name <span class="token operator">=</span> <span class="token string">"Rob"</span><span class="token punctuation">;</span> <span aria-hidden="true" class="line-numbers-rows"><span></span><span></span><span></span></span></code> </pre> </body></html>
